internal/backends/backends.go

```diff
@@ -27,6 +27,7 @@ import (
 var languageBackends = []api.LanguageBackend{
 	python.Python3Backend,
 	python.Python2Backend,
+	nodejs.NodejsBunBackend,
```

Comment:
How will the bun backend be distinguished from the node one? Is there a different language key?

Reply:
via the `Name` property on the `api.LanguageBackend`. It's set to `nodejs-bun`, which follows the other backend names `nodejs-npm` and `nodejs-yarn`

Reply:
although I think setting it to just `bun` might be better, since bun has its own runtime and may be used separately from node (ie, you can use bun as *just* a packager, or also as a runtime)

Reply:
renamed in latest commit